tools/src/check-packages/checkDependenciesAsync.ts

```diff
@@ -62,6 +57,21 @@ const SPECIAL_DEPENDENCIES: Record<string, Record<string, IgnoreKind | void> | v
     typescript: 'ignore-dev', // TODO: Should probably be a peer dep
   },
 
+  'expo-asset': {
+    '@expo/config-plugins/build/utils/warnings.js': 'ignore-dev', // TODO: Remove
+  },
+
+  'expo-updates': {
+    debug: 'ignore-dev', // TODO: Add dependency
+    getenv: 'ignore-dev', // TODO: Add dependency
+    '@expo/plist': 'ignore-dev', // TODO: Add dependency
+  },
```

Comment:
I've started adding TODOs here, but the package is still ignore since a sub-folder of it (`utils`) has a lot of invalid dependencies that'll need to be resolved separately